docs/Configuration.md

```diff
@@ -237,6 +237,41 @@ Jest will fail if:
 * The `./src/api/very-important-module.js` file has less than 100% coverage.
 * Every remaining file combined has less than 50% coverage (`global`).
 
+### `forceCoverageRegex` [string]
+
+Default: `''`
+
+The pattern to enforceCollecting coverage even if the file is in the ignored
```

Comment:
Can you rewrite this? What about "Test files are normally ignored from collecting code coverage. With this option, you can overwrite this behavior and include otherwise ignored files in code coverage."

Reply:
Sure thing. 👍